src/app/components/ErrorPage/__snapshots__/index.test.jsx.snap

```diff
@@ -47,10 +47,8 @@ exports[`ErrorMain should correctly render for an error page for News 1`] = `
   font-size: 1.75rem;
   line-height: 2rem;
   color: #3F3F42;
-  font-family: ReithSerif,Helvetica,Arial,sans-serif;
```

Comment:
These seem important

Reply:
umm! not sure, these were set in during snapshots update. lemme check

Reply:
@rhenshaw56 Howcome this conversation was marked as resolved? Its not been fixed

Reply:
@dr3 sorry, thought those were valid! fixed now